src/FSharpSpreadsheetML/Workbook.fs

```diff
@@ -87,4 +85,40 @@ module WorkbookPart =
         else 
             initSharedStringTablePart workbookPart
             |> getSharedStringTablePart
+    
+    let getSharedStringTable (workbookPart:WorkbookPart) =
+        workbookPart 
+        |> getSharedStringTablePart 
+        |> SharedStringTable.get
+
+    /// Returns the data of the first sheet of the given workbookpart
+    let getDataOfFirstSheet (workbookPart:WorkbookPart) = 
+        workbookPart
+        |> getWorkSheetParts
+        |> Seq.head
+        |> Worksheet.get
+        |> Worksheet.getSheetData
 
+    /// Either appends a new sheet with the given sheet data to the excel document, or, if the sheet already exists, updates it with the given sheet data
```

Comment:
The second part of the comment is wrong. This function will add a new sheet to the workbook in either case. Therefore athe preexistence of the given name should be checked